rekolekcje-webapp/src/app/core/store/parish/parish-reducer.ts

```diff
@@ -0,0 +1,45 @@
+import { Parish } from '../../../shared/models/parish.model';
+import { ParishSharedActions } from '../../../shared/store-shared/parish/parish-actions';
+
+export namespace Parishes {
+  export interface State {
+    parishList: Parish[];
+    parishesLoading: boolean;
+    selectedParish: Parish;
+  }
+
+  export const initialState: Parishes.State = {
+    parishList: [],
+    parishesLoading: false,
+    selectedParish: null,
+  };
+}
+
+export namespace ParishReducer {
+
+  export function reducer(state: Parishes.State = Parishes.initialState, action): Parishes.State {
+    switch (action.type) {
+
+      case ParishSharedActions.types.CreateParishFail: {
```

Comment:
This is not needed, as there is default for this switch code block, which always returns previous state